ReactWindows/ReactNative/Common/AccessibilityHelper.cs

```diff
@@ -0,0 +1,258 @@
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+
+namespace ReactNative.Common
+{
+    /// <summary>
+    /// Accessibility helper methods.
+    /// </summary>
+    public static class AccessibilityHelper
+    {
+        #region ImportantForAccessibility - Provides support for ImportantForAccessibility property implementation.
+
+        /// <summary>
+        /// Initializes the value of AccessibilityView for <paramref name="uiElement"/> and its children,
+        /// based on the ancestor <paramref name="parentUIElement"/> settings. Must be callded
```

Comment:
>callded [](start = 87, length = 7)

typo

Reply:
Fixed

---
In reply to: [164941941](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r164941941) [](ancestors = 164941941)